precompiles/parachain-staking/StakingInterface.sol

```diff
@@ -265,4 +265,23 @@ interface ParachainStaking {
         uint256 candidateAutoCompoundingDelegationCount,
         uint256 delegatorDelegationCount
     ) external;
+
+    /// @dev Fetch the total staked amount of a delegator, regardless of the
+    /// candidate.
+    /// @custom:selector e6861713
+    /// @param delegator Address of the delegator.
+    /// @return Total amount of stake.
+    function getDelegatorTotalStaked(address delegator)
+        external
+        view
+        returns (uint256);
+
+    /// @dev Fetch the total staked towards a candidate.
+    /// @custom:selector 670c59e9
+    /// @param candidate Address of the candidate.
+    /// @return Total amount of stake.
+    function getCandidateTotalBacked(address candidate)
```

Comment:
```suggestion
    function getCandidateTotalCounted(address candidate)
```
the implementation is using the `total_counted` which is the sum of the top delegations amounts + candidate self bond. It does not include the sum of bottom delegation amounts.

I don't see candidate.total_backed stored anywhere so either change the name as per suggestion or change the impl to return `CandidateInfo.total_counted + sum of BottomDelegations amounts`